codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -22,12 +23,20 @@ class SaveTermsAgreementInteractor(BaseInteractor):
     def validate(self, input: TermsAgreementInput):
         if input.terms_agreement is None:
             raise ValidationError("Terms of agreement cannot be null")
+        if input.customer_intent and input.customer_intent not in [
+            "Business",
```

Comment:
Any reason why we don't lowercase/uppercase the input to follow what we have in the DB?

Reply:
I am validating to keep parity with the [choices](https://github.com/codecov/codecov-api/blob/main/codecov_auth/migrations/0051_user_customer_intent.py) here.